public/components/datasources/components/manage/manage_data_connections_table.tsx

```diff
@@ -95,6 +99,7 @@ export const ManageDataConnectionsTable = (props: HomeProps) => {
       },
     ]);
     fetchDataSources();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I feel we want to add all dependencies, as otherwise it could potentially causes issues

Reply:
Probably. Since the lint issues have been there for a while and I [ran into issues](https://github.com/opensearch-project/dashboards-observability/blob/ce9db203e4922d0b3e83e5ce6f09b1e4c9f23a66/public/components/integrations/components/setup_integration.tsx#L253-L254) with adding deps to the hooks in the past, I decided to mark them ignored for now. Might be worthwhile to track a dedicated owner to look again at the lints here?